worlds/inscryption/Options.py

```diff
@@ -0,0 +1,116 @@
+from dataclasses import dataclass
+
+from Options import Toggle, Choice, DeathLinkMixin, PerGameCommonOptions
+
+
+class Act1DeathLinkBehaviour(Choice):
+    """If DeathLink is enabled, determines what counts as a death in act 1. This affects deaths sent and received.
+    Sacrificed: Send a death when sacrificed by Leshy. Receiving a death will extinguish all candles.
+    Candle Extinguished: Send a death when a candle is extinguished. Receiving a death will extinguish a candle."""
+    auto_display_name = True
```

Comment:
This is True by default with `Choice` options and can be removed from all Choice options here
```suggestion
```